### PR TITLE
Fixed Error in example pubspec and Removed ? from WidgetsBinding

### PR DESCRIPTION
**Pull request by jabguru**
The package shows an error in Flutter 3.0 because of the null aware operator on WidgetsBinding.instance.
Also, I fixed the error shown by linter in pubspec.yaml for the example.

### CODE REVIEW

File: example/ios/Flutter/flutter_export_environment.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```
Comment (jaumard) [inline example/ios/Flutter/flutter_export_environment.sh]: Shouldn't be here, can you please remove it

**Review by jaumard — CHANGES_REQUESTED** [review]
Hey thanks for the PR! 
Few little changes needed please

File: example/ios/Flutter/flutter_export_environment.sh (hunk 1 above)
Comment (jaumard) [reply example/ios/Flutter/flutter_export_environment.sh]: I was able to do it myself all good :)

**Comment by jabguru** [comment]
Thanks!

**Pull request merged**